exercises/robot-simulator/defs.go

```diff
@@ -19,14 +19,14 @@ type Command byte // valid values are 'R', 'L', 'A'
 type RU int
 type Pos struct{ Easting, Northing RU }
 type Rect struct{ Min, Max Pos }
-type DirAt struct {
+type Robot struct {
 	Dir
 	Pos
 }
 
 // additional definition used in step 3
 
-type Place struct {
+type Robot3 struct {
```

Comment:
I'll agree that `Place` was not a good name for it. What would you say about `NamedRobot` so that from reading the name we can get an idea of how it differs from `Robot`?